### DISCUSSION TITLE
Add macro support for executing on each project

### DISCUSSION THREAD

**Discussion by praveer-k** [alefragnani/vscode-project-manager#664] [Ideas]
Possible application of it could be:
1. git pull on each project.
2. check stats for each project.
etc.

**Comment by alefragnani** [comment]
Hi @praveer-k ,

I'm not sure what do you expect as a _macro_ (run VS Commands or command line batches), but I think this is _out of scope_ for the Project Manager extension.

I remember someone, a few years ago, asking for _run commands_ when a project is opened. By that time, I suggested to take a look at [Auto Run Commad](https://marketplace.visualstudio.com/items?itemName=gabrielgrinberg.auto-run-command) extension. Maybe it will work for you as well.

Hope this helps
